### PR TITLE
community/glide: upgrade to v0.13.1

### PR DESCRIPTION
**Pull request by leprechau**
Updated glide to latest release and added check function to execute upstream tests.

### CODE REVIEW

File: community/glide/APKBUILD [hunk 1]
```diff
@@ -26,9 +26,15 @@ build() {
 	go build -o glide -ldflags "-X main.version=$pkgver" glide.go || return 1
```
Comment (TBK) [inline community/glide/APKBUILD]: Commit (13e67c291ac92df3239e6bcd7dc76eeae96e5d2e) by @rgdoliveira upgraded Glide to 0.13.1.

So this PR could be updated to add the check.

` || return 1` should be removed.

**Comment by clandmeter** [comment]
I'm closing this as its stale. If somebody want to add a check please create a new PR.

**Comment by leprechau** [comment]
Not really sure what the problem was with this PR ... did the package ever get updated?

**Comment by leprechau** [comment]
Nevermind I see the referenced commit for the update.  If nobody else is concerned about the lack of tests I'm good with letting it die.

**Pull request closed without merging**